sql/2020/22_HTTP_2/avg_number_of_h2_pushed_resources_and_avg_bytes.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+# 22_HTTP_2 - Average number of HTTP/2 Pushed Resources and Average Bytes
+SELECT 
+  client,
+  COUNT(DISTINCT page) AS num_pages,
+  ROUND(AVG(num_requests),2) AS avg_pushed_requests,
+  ROUND(AVG(kb_transfered),2) AS avg_kb_transfered
+FROM (
+
+SELECT 
```

Comment:
Nit: whitespace
```suggestion
  SELECT
```

Reply:
Understood. What I did was groom all files to remove whitespace between any SQL statements. However, for readability I decided to leave whitespace between FUNCTION and SQL code. I also left whitespace before and after UNION ALL statement, agin for easier readability. Let me know if you disagree. If so i can remove whitespace from ALL files regardless of code.